3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -158,3 +107,5 @@ if [ "${MEMBER_ID}" = '0' ]; then
 else
   add_member "${MEMBER_HOST}"
 fi
+
+>/tmp/initialized
```

Comment:
It is probably feasible to do so, but in practice I think all the code using k8s replicasets should be deprecated/removed from this repo - I can't think of a reason why anyone would/should use it now that statefulsets are available.  Indeed this repo doesn't even actually ship a template which enables their use - the only one was the mongodb-clustered.json under 2.4.